test/models/friend_request_test.rb

```diff
@@ -0,0 +1,41 @@
+require 'test_helper'
+
+class FriendRequestTest < ActiveSupport::TestCase
```

Comment:
Let us also add tests for our instance private methods `not_pending` and `not_friends`